cli/cmd/stat.go

```diff
@@ -389,3 +411,53 @@ func sortStatsKeys(stats map[string]*row) []string {
 	sort.Strings(sortedKeys)
 	return sortedKeys
 }
+
+// Perform all validation on the command-line options, returning the first
+// error encountered, or `nil` if the options are valid.
+func (o *statOptions) validate(resourceType string) error {
+	err := o.validateConflictingFlags()
+	if err != nil {
+		return err
+	}
+
+	if resourceType == k8s.Namespace {
+		err := o.validateNamespaceFlags()
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// Validate that the options do not contain mutually exclusive flags.
+func (o *statOptions) validateConflictingFlags() error {
+	if o.toResource != "" && o.fromResource != "" {
+		return errors.New("--to and --from flags are mutually exclusive")
```

Comment:
Throughout the rest of this file we use `fmt.Errorf`, which is equivalent to `errors.New`, and my preference is to use the `fmt` package throughout, rather than adding an import for the `errors` package.

Reply:
Okay, will do. I originally used `fmt.Errorf`, but figured it was better to not do formatting if there was nothing to format? Will switch it back.